src/include/main/client_context.h

```diff
@@ -109,8 +109,9 @@ class KUZU_API ClientContext {
     // Query.
     std::unique_ptr<PreparedStatement> prepare(std::string_view query);
     std::unique_ptr<QueryResult> executeWithParams(PreparedStatement* preparedStatement,
-        std::unordered_map<std::string, std::unique_ptr<common::Value>> inputParams);
-    std::unique_ptr<QueryResult> query(std::string_view queryStatement);
+        std::unordered_map<std::string, std::unique_ptr<common::Value>> inputParams,
```

Comment:
ditto I would not change this API given we don't have multi query in a prepared statement.